src/goto-instrument/contracts/dynamic-frames/dfcc_loop_nesting_graph.h

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************\
+
+Module: Dynamic frame condition checking
+
+Author: Qinheping Hu, qinhh@amazon.com
+Author: Remi Delmas, delmasrd@amazon.com
+
+\*******************************************************************/
+
+/// \file
+/// Builds a graph describing how loops are nested in a GOTO program.
+
+#ifndef CPROVER_GOTO_INSTRUMENT_CONTRACTS_DYNAMIC_FRAMES_DFCC_LOOP_NESTING_GRAPH_H
+#define CPROVER_GOTO_INSTRUMENT_CONTRACTS_DYNAMIC_FRAMES_DFCC_LOOP_NESTING_GRAPH_H
+
+#include <util/graph.h>
+
+#include <goto-instrument/loop_utils.h>
+
+class messaget;
+
+/// A graph node that stores information about a natural loop.
+struct dfcc_loop_nesting_graph_nodet : public graph_nodet<empty_edget>
+{
+public:
+  dfcc_loop_nesting_graph_nodet(
+    goto_programt::targett head,
+    goto_programt::targett latch,
+    const loopt instructions)
+    : head(head), latch(latch), instructions(instructions)
+  {
+  }
+
+  /// Loop head instruction
+  goto_programt::targett head;
+
+  /// Loop latch instruction
+  goto_programt::targett latch;
+
+  /// Set of loop instructions
+  const loopt instructions;
```

Comment:
Let's try to explicitly use `const typename natural_loops_mutablet::loopt &instructions` to see if we can resolve the build error.

Reply:
fixed